docs/awscli/index.rst

```diff
@@ -1,3 +0,0 @@
-AWS CLI
```

Comment:
Removing for now, can readd this when we add `aws` security (i.e. a cloudformation deploy shim).  On the roadmap